### DISCUSSION TITLE
Does WPF do the Chromium WebView2 Control and .NET to JavaScript Interop

### DISCUSSION THREAD

**Discussion by taversnack** [ChromiumDotNet/WebView2.DevTools.Dom#19] [General]
Hiya,
I've been looking around for some time and came across this, i was wondering in the WPF example does that do the WebView2 control .net to Javascript interpolation by any chance?

**Comment by amaitland** [comment]
What do you mean exactly?

**Reply by taversnack** [reply]
I've been looking at this article: https://weblog.west-wind.com/posts/2021/Jan/26/Chromium-WebView2-Control-and-NET-to-JavaScript-Interop-Part-2

Is this similar to what you are doing for the WPF sample?

**Reply by amaitland** [reply]
See https://github.com/ChromiumDotNet/WebView2.DevTools.Dom#webview2-devtools-dom for a summary

> Is this similar to what you are doing for the WPF sample?

The project uses the chrome devtools protocol. See https://chromedevtools.github.io/devtools-protocol/

**Reply by taversnack** [reply]
So it doens't i would assume
